src/interpreter.js

```diff
@@ -607,14 +607,35 @@ export function getRangeTargetElements(
   );
 
   // Get the start and end character offsets
-  const startOffset = parseInt(CFIAST.cfiString.range1.termStep.offsetValue, 10) || undefined;
```

Comment:
According to a spec: "there should be no character offset data" in the case where the CFI range is made up of just elements, and no text nodes.

https://github.com/evidentpoint/readium-cfi-js/blob/c4c0724625cb58aa91d4d53f77716165d9123764/test/specs/interpreter.spec.js#L400

Reply:
Failing test:

```
CFI INTERPRETER OBJECT range CFI interpretation can return target elements when the targets are different elements FAILED
	Expected 0 to be undefined.
	<Jasmine>
	./specs/interpreter.spec.js/</</<@dist/bundle.js:18888:37
	<Jasmine>
	Expected 0 to be undefined.
	<Jasmine>
	./specs/interpreter.spec.js/</</<@dist/bundle.js:18889:35
	<Jasmine>

Reply:
I have modified the commit to allow returning undefined offsets. It should be noted that offsets of 0 are returned as undefined so it is up to the caller to handle that case.